config/config_params.go

```diff
@@ -119,6 +119,7 @@ type Config struct {
 
 	ChainInsertMode             string `config:"oneof(insert,append);insert;non-zero,die-on-fail"`
 	DefaultEndpointToHostAction string `config:"oneof(DROP,RETURN,ACCEPT);DROP;non-zero,die-on-fail"`
+	DefaultForwardChainAction   string `config:"oneof(RETURN,ACCEPT);ACCEPT;non-zero,die-on-fail"`
```

Comment:
How about renaming this to `IptablesAllowAction`; the `DefaultEndpointToHostAction` above controls the default when there is no policy whereas this new option control what happens when the policy allows a packet.